src/deps_info.json

```diff
@@ -50,9 +50,22 @@
   "emscripten_GetAlcProcAddress": ["strcmp"],
   "emscripten_GetAlProcAddress": ["strcmp"],
   "emscripten_get_preloaded_image_data_from_FILE": ["fileno"],
+  "__gxx_personality_v0": ["_ZSt18uncaught_exceptionv", "__cxa_find_matching_catch"],
+  "__cxa_find_matching_catch": ["__cxa_is_pointer_type", "__cxa_can_catch"],
+  "__cxa_find_matching_catch_0": ["__cxa_is_pointer_type", "__cxa_can_catch"],
```

Comment:
why are all these 1,2,3 etc. versions needed? and why just 10?

Reply:
The change to explicitly list out `__cxa_find_matching_catch_<number>` is a change to migrate these functions to be defined as JS library functions (in [src/library_exceptions.js](https://github.com/juj/emscripten/blob/c239dbe738e294e466eaf3daa2263dd555bc4e1d/src/library_exceptions.js#L319) ), instead of the symbols being defined ad hoc in JS linker (used to be in src/jsifier.js at https://github.com/emscripten-core/emscripten/blob/ee231b04f67de484e4e1dcd4200d538f6b5722c4/src/jsifier.js#L103). See https://github.com/emscripten-core/emscripten/issues/8046 that discusses this aspect in more detail.

Reply:
I see, sounds good - if we have a nice error if we need more than 10. What would happen in that case?

Reply:
if there are more than 10, there will likely be an undefined symbol error, since the linker cannot find a function that LLVM emitted a dependence to

Reply:
I worry that's not super-clear, but if it comes up we can look into improving it I guess.